### DISCUSSION TITLE
ClickAway 관련 - zIndex

### DISCUSSION THREAD

**Discussion by healtheloper** [Co-Studo/Co-Studo-front#14] [Ideas]
#13 

@mina-gwak : click away 가 생성이 되면
만약 Popover 인 경우 안에 있는 내용을 드래깅 한다던지 혹은 Input 을 입력할 수도 있는데
그 상황에도 Open 관련 상태가 바뀔 수 있지 않을까?

@healtheloper : 충분히 가능성 있어 보이고 Popover 같은 컴포넌트를 만들면서 이슈가 생길 때 대응해보자

**Comment by jindonyy** [comment]
제 생각엔 가상선택자로 넣어놔서 문제 없을거 같기도 하네여 🤔

**Reply by mina-gwak** [reply]
저도 그럴 줄 알았는데 아까 테스트해보니 드롭다운 내부를 클릭해도 닫히더라구요!
로그아웃 메뉴처럼 바로 닫혀도 되는 건 상관 없는데 저런 경우에는 문제가 될 수 있을 것 같아요
